api/migrations/0002_data_migration.py

```diff
@@ -0,0 +1,112 @@
+import datetime
+
+from django.db import migrations
+from django.db.backends.sqlite3.schema import DatabaseSchemaEditor  # for typing only
```

Comment:
You can remove these comments about  typing :)

Reply:
OK :)

Reply:
I will also replace `sqlite3` with `postgresql` to be clear that that's what we'll be using in production